React/React Guidelines and Best Practices (outdated).md

```diff
@@ -0,0 +1,676 @@
+# React Guidelines and Best Practices
+
+*Guides are not rules and should not be followed blindly. Use your head and think.*
+
+### File organization and naming
```

Comment:
Some projects, like Underline, have adopted the concept of [Atomic components](https://bradfrost.com/blog/post/atomic-web-design/). It seems to be working pretty well for them. Do we want to make a mention of it here as well?

Reply:
Yes, this is something that @danijelbuhin is currently working on